map/traffic_manager.cpp

```diff
@@ -120,8 +122,10 @@ void TrafficManager::Clear()
 {
   m_currentCacheSizeBytes = 0;
   m_mwmCache.clear();
-  m_lastMwmsByRect.clear();
-  m_activeMwms.clear();
+  m_lastDrapeMwmsByRect.clear();
+  m_lastRoutingMwmsByRect.clear();
```

Comment:
Can't say that I like this quite specific approach, but I can't block this PR. Leave as is if you wish.